<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewAdminInfo</class>
 <widget class="QDialog" name="viewAdminInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{font-family:century gothic;
color:white;
}
QLabel{
font-size:16px;
}
#head{
border-bottom-left-radius: 20px;
border-bottom-right-radius: 20px;
background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #ad5339, stop:1 #3c1053);
font-size:24px;
font-weight:bold;
}
QLineEdit{
font-size:16px;
font-weight:bold;
color:black;
background:white;
}
#viewAdminInfo{
background:#282828;
}
QPushButton {
font-size:15px;
font-weight:bold;
border:none;
border-radius:5px;
background: #ff0303;
}
QPushButton:hover {
background:#49ebff;
color:black;
}</string>
  </property>
  <widget class="QLabel" name="lastNameText">
   <property name="geometry">
    <rect>
     <x>436</x>
     <y>60</y>
     <width>208</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="contactInfoText">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>170</y>
     <width>106</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Contact Info </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dateOfHiring">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>401</width>
     <height>28</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>mm/dd/yy</string>
   </property>
  </widget>
  <widget class="QLabel" name="dateOfHiringText">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>250</y>
     <width>122</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Date Of Hiring   </string>
   </property>
  </widget>
  <widget class="QLabel" name="emailaddressText">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>210</y>
     <width>119</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Email Address   </string>
   </property>
  </widget>
  <widget class="QLabel" name="head">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>0</y>
     <width>660</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>VIEW ADMIN INFO</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>261</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="firstName">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>90</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lastName">
   <property name="geometry">
    <rect>
     <x>436</x>
     <y>90</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="usernameText">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>290</y>
     <width>78</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="firstNameText">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>60</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>First Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="birthdayText">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>130</y>
     <width>154</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Birthday mm/dd/yy </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>262</width>
     <height>28</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailaddress">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>399</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="contactInfo">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>400</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="passwordText">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>330</y>
     <width>72</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="middleNameText">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>209</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Middle Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>372</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="middleName">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="birthday">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>399</width>
     <height>28</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>mm/dd/yy</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
